### PR TITLE
[Guide - The New Architecture] Fabric Components as Native Components

### PR DESCRIPTION
**Pull request by cipolleschi**
This PR adds the guide to create a backward compatible Fabric Component.

Draft while we wait for Android.

### CODE REVIEW

**Comment by cortinico** [comment]
> Draft while we wait for Android.

Same as the other PR, can I push here or should I do a separate PR?

**Comment by cipolleschi** [comment]
Yes, push here!

File: docs/the-new-architecture/backward-compatibility-fabric-components.md [hunk 1]
```diff
@@ -3,10 +3,428 @@ id: backward-compatibility-fabric-components
...
 
-- How to avoid installing dependencies when they are not needed
-- The usage of compilation pragmas to avoid compiling code that requires types from the codegen
-- API uniformity in JS, so that they don’t have to import different files
+Fabric Components only work when the New Architecture is properly setup. If you already have a library that you want to migrate to the New Architecture, have a look at the [migration guide](../new-architecture-intro) as well.
+:::
+
+Creating a backward compatible Fabric Component lets your users continue leverage your library, independently from the architecture they use. The creation of such a component requires a few steps:
+
+1. Configure the library so that dependencies are not installed in the old architecture.
```
Comment (cortinico) [inline docs/the-new-architecture/backward-compatibility-fabric-components.md:19]: Same as for the other PR (I would omit duplicated comments so we resolve them only once)

File: docs/the-new-architecture/backward-compatibility-fabric-components.md [hunk 2]
```diff
@@ -3,10 +3,428 @@ id: backward-compatibility-fabric-components
...
+export default myComponent;
+```
+
+</TabItem>
+</Tabs>
+
+Whether you are using Flow or TypeScript for your specs, we understand which architecture is running by checking if the `global.nativeFabricUIManager` object has been set or not.
+
+:::caution
+The `global.nativeFabricUIManager` API may change in the future for a function that encapsulate this check.
```
Comment (cortinico) [inline docs/the-new-architecture/backward-compatibility-fabric-components.md:424]:
```suggestion
Please note that the New Architecture is still experimental. The `global.nativeFabricUIManager` API might change in the future for a function that encapsulate this check.
```

**Review by cortinico — APPROVED** [review]
LGTM 👍 I haven't re-applied some of the comments I left on the other PR but they should be handled by both PRs

**Pull request merged**